src/draw/handler/Draw.Polyline.js

```diff
@@ -227,6 +241,24 @@ L.Draw.Polyline = L.Draw.Feature.extend({
 		this._mouseDownOrigin = null;
 	},
 
+	_onTouch: function (e) {
+		// #TODO: use touchstart and touchend vs using click(touch start & end).
+		if (L.Browser.touch) { // #TODO: get rid of this once leaflet fixes their click/touch.
```

Comment:
Idk, it's been awhile since I've looked or tested that. But from what I can
recall, I think it was because the mousedown and mouseup weren't matched
well with touchstart and touchend.

On Thu, Mar 26, 2015 at 12:25 PM, mrusful <notifications@github.com> wrote:

> In src/draw/handler/Draw.Polyline.js
> <https://github.com/Leaflet/Leaflet.draw/pull/285#discussion_r27247027>:
>
> > @@ -227,6 +241,24 @@ L.Draw.Polyline = L.Draw.Feature.extend({
> >  		this._mouseDownOrigin = null;
> >  	},
> >
> > +	_onTouch: function (e) {
> > +		// #TODO: use touchstart and touchend vs using click(touch start & end).
> > +		if (L.Browser.touch) { // #TODO: get rid of this once leaflet fixes their click/touch.
>
> Can you clarify what exactly leaflet should fix?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Leaflet/Leaflet.draw/pull/285/files#r27247027>.
>



-- 
Michael Guild
571.338.9782
michaelguild13@gmail.com